cmd/sops/main.go

```diff
@@ -429,6 +429,7 @@ func main() {
 
 		unencryptedSuffix := c.String("unencrypted-suffix")
 		encryptedSuffix := c.String("encrypted-suffix")
+		encryptedSelector := c.String("encrypted-selector")
```

Comment:
You need to add the flag to the list of flags here: https://github.com/mozilla/sops/blob/457297faa309c5a0b4536ef5c808b5324e025fb6/cmd/sops/main.go#L409